primer/src/Primer/Eval/Redex.hs

```diff
@@ -224,7 +232,7 @@ data Redex
       , varID :: ID
       -- ^ Where was the occurrence (used for details)
       }
```

Comment:
Maybe I should revisit this definition, since recording these details seems slightly silly now they are all local. Similarly for `InlineLetRec` and `InlineLetInType`

Reply:
Actually, on second thought I don't think we should. These definitions are all internal, and the implementation of `runRedex{,Ty}` is nicer if we do not need to pattern match to find these identifiers.

The reason we need these IDs is to expose them in the "details" over the API, where I think we should still have them for the benefit of dumb frontends.